sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -2573,20 +2568,22 @@
       }
     },
     "TwinProperties": {
-      "description": "Represents Twin properties",
+      "description": "Represents twin's desired and reported properties. The maximum depth of twin property objects is 10.",
       "type": "object",
       "properties": {
         "desired": {
-          "description": "Used in conjunction with reported properties to synchronize device configuration or condition. Desired properties can only be set by the solution back end and can be read by the device app. The device app can also be notified in real time of changes on the desired properties.",
+          "description": "Properties that set by the solution back end and read by the device.",
```

Comment:
This is talking of the service implementation. We should mention what this property is.

Reply:
This is talking about `"desired"` property.

Reply:
1) "Properties that **are** set by ..."
2) Can you explain what is solution back end? Is this read only in that case?

Reply:
solution back-end here would be any application doing service-side operations. No, they wouldn't be readonly, infact, from service client perspective, those would be writeable.